docs(tutorial): Material UI option (#5672)
refinedev/refine: documentation/tutorial/next-steps/cli/react-router/material-ui/index.md

- `update` command to check and update versions of the Refine packages,
- `swizzle` command to export Refine components and providers on your project to customize them.

## Installation

At the beginning of the tutorial, we've already installed the `@refinedev/cli` but if you've missed this step, you can use the below command to install it easily:

<InstallPackagesCommand args="@refinedev/cli" />

## Using Runners

We've already handed over our runners to the CLI's runners, our `package.json` file's scripts should look like the below:

```json title="package.json"
{
  "scripts": {
    "dev": "refine dev",
    "build": "refine build",
    "start": "refine start",
    "refine": "refine"
  }
}
```

These commands will use the same runners as your bundler, and using them is optional but recommended for staying updated with Refine team's updates and receiving notifications about new package versions.

:::note

Notice that we have the `refine` command defined in `"scripts"`. This command is used to run the CLI commands. You can use the `refine` command to run the CLI commands with the `npx` command as well.

:::

## Using the `add` Command

> The `add` command creates new resources and providers with a single command.

Reviewer: ```suggestion
The `add` command can be used to create resources and providers.
```

Try to run the following command to create a new resource called `categories`:

```sh
npm run refine add resource categories
```

This command will create files to an appropriate location in your project, populates them with Inferencer components, and add the resource definition to your `<Refine />` component.

<VideoInView src="https://refine.ams3.cdn.digitaloceanspaces.com/assets/tutorial/webm/add-resource.webm" playsInline loop autoPlay muted />

> We can use the `add` command to create a new provider, with available provider options including:

Reviewer: ```suggestion
We can use the `npm run refine add provider` command to create multiple providers, with available provider options including:
```

- Auth Provider to handle authentication,
- Data Provider to handle data fetching and mutations,
- Live Provider to handle real-time updates,
- Access Control Provider to handle access control and permissions.
- Audit Log Provider to submit/list audit logs.
- i18n Provider to handle internationalization and localization.
- Notification Provider to handle notifications.

Try to run the following command to create a new i18n Provider:

```sh

> npm run refine add i18n

Reviewer: ```suggestion
npm run refine add provider i18n
```
